tests/test_evaluation.py

```diff
@@ -39,39 +39,24 @@ def pred():
     )
 
 
-def test_score_df(data, pred):
+def test_score_df(data, pred, score_fun):
     """
-    Return the expected forecast start, end and correct MSPE.
+    Return the expected forecast start, end, and correct MSPE.
     """
     data = iup.IncidentUptakeData(data)
     pred = iup.PointForecast(pred)
 
-    output = eval.score(data=data, pred=pred, score_fun=eval.mspe)
+    output = eval.score(data=data, pred=pred, score_fun=score_fun)
```

Comment:
How does the argument `score_fun` get its value? I don't see it assigned anywhere, nor do I see a default value assigned in `def score` in `iup/eval.py`. Maybe you need an `@pytest.fixture` for `score_fun` to pass it into your test this way?

Reply:
This is a good catch! The bug has been fixed.